### PR TITLE
Fix Wizard position in docs

### LINKED ISSUE #462: DOC: Fix Wizard position in docs to be consistent with the Solidity library doc site

### PR DESCRIPTION
**Pull request by ericnordelo**
Fixes #462. Moving the Wizard nav element right below Overview, and showing it in the same page from embedding script (as is done in Solidity contracts docs site).

### CODE REVIEW

**Comment by martriay** [comment]
It was a conscious decision of mine to move it at the very bottom, since I thought it added noise in an already hard to approach doc.

Basically, I didn't want people to go from overview directly into wizard, without understanding the extensibility pattern first (which unlike Solidity, is not as obvious as "just inherit".

image

That being said I recognize the inconsistency, so I'll just explain my rationale to better discuss what's better. Interested in @frangio's thoughts.

**Comment by martriay** [comment]
I just edited the description to make clear which issue is being addressed.

**Comment by martriay** [comment]
The preview is broken :c

**Comment by frangio** [comment]
@martriay Preview is broken because the branch isn't built on the latest `main`, and doesn't include #459.

**Comment by frangio** [comment]
Looks good now. If you have concerns about people understanding the extensibility pattern first, you could include text or even an "Impotant" callout box before the embed.

File: docs/modules/ROOT/pages/wizard.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+= Cairo Contracts Wizard
+:page-notoc:
+
+Not sure where to start? Use the interactive generator below to bootstrap your
+contract and learn about the components offered in OpenZeppelin Cairo Contracts.
+
+
+CAUTION: If you haven’t yet, before moving forward, we strongly recommend checking the xref:extensibility.adoc[Extensibility Pattern] section to understand how we manage extensibility in a language lacking inheritance.
```
Comment (martriay) [inline docs/modules/ROOT/pages/wizard.adoc:8]:
```suggestion
NOTE: We strongly recommend checking the xref:extensibility.adoc[Extensibility Pattern] to understand how to extend from our library.
```
i don't think we need to use caution in here

**Review by martriay — APPROVED** [review]

**Comment by ericnordelo** [comment]
Is this a change we should merge before releasing @martriay?

**Comment by martriay** [comment]
ideally by not critical

**Review by martriay — APPROVED** [review]

**Pull request merged**